@navikt/core/css/action-menu.css

```diff
@@ -0,0 +1,228 @@
+/* stylelint-disable csstools/value-no-unknown-custom-properties */
+/* stylelint-disable aksel/design-token-exists */
+.navds-action-menu__content {
+  --__ac-action-menu-content-p: 0.5rem;
+  --__ac-action-menu-item-pr: var(--a-spacing-3);
+  --__ac-action-menu-item-pl: var(--a-spacing-2);
+  --__ac-action-menu-item-height: 32px;
+
+  padding: var(--__ac-action-menu-content-p);
+  box-shadow: var(--a-shadow-medium);
+  border-radius: var(--a-border-radius-large);
+  background-color: var(--a-surface-default);
+  overflow: auto;
```

Comment:
Overflow auto is needed in these cases:
with
<img width="144" alt="Screenshot 2024-09-13 at 11 39 31" src="https://github.com/user-attachments/assets/a19dbaf4-a53b-464c-a936-e6e17038d1fe">
without
<img width="158" alt="Screenshot 2024-09-13 at 11 39 38" src="https://github.com/user-attachments/assets/c8298ff7-8fbe-4226-b24a-a0bc50aab039">


Nice call with the max-width, added it + removed white-space: no-wrap

Reply:
Maybe better demo of overflow auto
<img width="368" alt="Screenshot 2024-09-13 at 11 47 57" src="https://github.com/user-attachments/assets/6a1376c9-2379-4777-8b3c-46a8f4d0e3df">

Reply:
![image](https://github.com/user-attachments/assets/38c670f3-2ff3-4c51-806b-0896babd975f)
for me this demo wraps a bit (probably not intentional?), if min-width is large enough it's fine.

Reply:
Testing now in FF, also this disabled example looks a bit strange when the buttons appear to be clickable (but they're not actually clickable) 🤔 
![image](https://github.com/user-attachments/assets/5479929d-2f17-4923-a3bc-817cc1a1af81)

Reply:
it looks ok in Edge/Chrome for me

Reply:
> Testing now in FF, also this disabled example looks a bit strange when the buttons appear to be clickable (but they're not actually clickable) 🤔 ![image](https://private-user-images.githubusercontent.com/819074/367275287-5479929d-2f17-4923-a3bc-817cc1a1af81.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMjcyNzAsIm5iZiI6MTcyNjIyNjk3MCwicGF0aCI6Ii84MTkwNzQvMzY3Mjc1Mjg3LTU0Nzk5MjlkLTJmMTctNDkyMy1hM2JjLTgxN2NjMWExYWY4MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkxM1QxMTI5MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03ZDBkMDMwYTRjMDY5OTAyOTdjYTQ3ZGI5OWI4ZjU3MTU1MGViOGVmM2JjNDUyMzFjM2NlZTI3YzNmN2U4ZGJlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.85eN8emsxDT1m0WZooVbhbD1PPL_3K0OUeomOo202Bw)

nvm this, it's seemingly supposed to be this way, but normally you wouldn't mix disabled & enabled checkboxes & radio, just for display purposes right?

Reply:
Is related to the scrollbar showing up, looks ok without scrollbar. Can be fixed by adding `scrollbar-gutter: stable;`, but not a fan of that solution.

Reply:
How to reproduce? Tried using FF and making the window smaller so that I get a scrollbar.

Reply:
?path=/story/ds-react-actionmenu--checkbox-groups 

Uploading Screen Recording 2024-09-13 at 13.53.37.mov…

Reply:
Have you tried removing `scrollbar-width: thin`?

Could this work? `width: calc(100% + 10px)` (replace 10px with width of scrollbar)

Reply:
`width: calc(100% + 17px);` worked like charm! 🎉  17px seems to be the max in all browsers, while some are less. 
Ended ut removing `scrollbar-width: thin` because of a11y concerns.